src/spec/operations.ts

```diff
@@ -67,11 +67,29 @@ function getOperationResponses(op: any): ApiOperationResponse[] {
 }
 
 function getOperationSecurity(op: any): ApiOperationSecurity[] {
-  if (!op.security || !op.security.length) return
+  if (!op.security) return;
 
-  return op.security.map(def => {
-    const id = Object.keys(def)[0]
-    const scopes = def[id].length ? def[id] : undefined
-    return { id, scopes }
-  })
+  if (op.security.length) {
+    return op.security.map(def => {
+      const id = Object.keys(def)[0];
+      const scopes = def[id].length ? def[id] : undefined;
+      return { id: id, scopes: scopes };
+    });
+  } else if (Object.keys(op.security).length) {
```

Comment:
I couldn't spot in the [spec](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md) where this would be an object and not an array, but maybe I missed something. Could you clarify this one?

Reply:
Oops, I misread the spec here. I'll change this in a push shortly.